src/cdk/drag-drop/drop-list-container.ts

```diff
@@ -7,9 +7,13 @@
  */
 
 import {InjectionToken, QueryList, ElementRef} from '@angular/core';
-import {CdkDrag} from './drag';
+import {CdkDrag} from './directives/drag';
 
 
+/**
```

Comment:
Could you add a comment above here that explains the background on _why_ this is being removed?